features/show_menu.feature

```diff
@@ -0,0 +1,21 @@
+Feature: let_custemers_see_menu
+as a owner.
+to be able to let costumers buy food.
+You want to display a Menu on a page.
```

Comment:
Please use natural language (see feature title) and follow the User story format.